ci-operator/step-registry/hypershift/kubevirt/create/disconnected/workarounds/hypershift-kubevirt-create-disconnected-workarounds-commands.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+# TODO: do here only the preliminary workarounds
+# so that we can have the cleanest possible set of
+# changes in the hypershift-kubevirt-create step
+
+set -exuo pipefail
+
+source "${SHARED_DIR}/packet-conf.sh"
+
+scp "${SSHOPTS[@]}" "/etc/quay-pull-credentials/registry_quay.json" "root@${IP}:/home/registry_quay.json"
+
+MCE=${MCE_VERSION:-""}
+echo "$MCE" > /tmp/mce
+scp "${SSHOPTS[@]}" "/tmp/mce" "root@${IP}:/home/mce"
+
+# shellcheck disable=SC2087
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << 'EOF'
```

Comment:
For non-file content(REDHAT_OPERATORS_INDEX_TAG,CCS_OPERATOR_CHANNELS, etc), we can accomplish it by passing variables, while for file content(${CLUSTER_PROFILE_DIR}/pull-secret), we can still use SCP.，E.g:
https://github.com/openshift/release/blob/4618e802f1a46035bf84ac3dbc34a01b443d9f45/ci-operator/step-registry/hypershift/mce/install/disconnected/hypershift-mce-install-disconnected-commands.sh#L16